### DISCUSSION TITLE
Remotely save 在iOS上频繁让obsidian重启

### DISCUSSION THREAD

**Discussion by wanderrover** [remotely-save/remotely-save#669] [Q&A] [answered]
这个现象已经有好一段时间了，之所以说是remotely save的引起的是我做了实验，如果关掉rs，那么程序就不会重启，如果手动或者自动同步，那么OB一定会重启，而且是每次同步都会重启。但是确实同步内容是同步到移动端上了。
不知道大佬知道不知道这个是什么原因引起的？

**Comment by fyears** [comment, marked as answer]
设置一下跳过大文件，手机上有性能问题

**Reply by wanderrover** [reply]
感谢！我测试一下

**Reply by wanderrover** [reply]
确实，问题得以解决，反正选了100M的那个选项是没问题的
